common/src/main/java/tachyon/network/protocol/RPCMessage.java

```diff
@@ -50,6 +50,15 @@ public void encode(ByteBuf out) {
     }
 
     /**
+     * Returns the int identifier of the type.
+     *
+     * @return the int id of the type
+     */
+    public int getId() {
+      return mId;
```

Comment:
It seems the only use of this getId() is to get an int from an enum and assign it to `DataServerMessage` as the message type. This agreement of conversion between `DataServerMessage.mRPCMessageType` and enum is (1) implicitly and (2) not really documented, which is risky to me.

I guess the reason to have this int<->PRCMessage.Type conversion is because RPCMessage is only available in worker, while DataServerMessage is defined in common, is it making sense to also move RPCMessage to common? Anyway RPC_BLOCK_REQUEST and RPC_BLOCK_RESPONSE seems to be needed by both client and server.